Test `testFicgs (remotegame.RemoteGameTestCase)` started failing

Hi @gbtami,

I checked if pull request #2398 was causing this somehow, but I'm afraid it's not (i.e. a branch with this reverted fails the same way):

```
2025-10-09T12:55:50.1117152Z FAIL: testFicgs (remotegame.RemoteGameTestCase)
2025-10-09T12:55:50.1124654Z ----------------------------------------------------------------------
2025-10-09T12:55:50.1125331Z Traceback (most recent call last):
2025-10-09T12:55:50.1134157Z   File "/home/runner/work/pychess/pychess/testing/remotegame.py", line 635, in testFicgs
2025-10-09T12:55:50.1134822Z     self.executeTest(InternetGameFicgs(), links)
2025-10-09T12:55:50.1135492Z   File "/home/runner/work/pychess/pychess/testing/remotegame.py", line 79, in executeTest
2025-10-09T12:55:50.1135957Z     self.assertEqual(ok, expected)
2025-10-09T12:55:50.1136208Z AssertionError: False != True
```

What do you see as the best way forward?

Best, Sebastian

--- a/lib/pychess/Savers/remotegame.py
+++ b/lib/pychess/Savers/remotegame.py
@@ -2235,32 +2235,6 @@ class InternetGameChesspro(InternetGameInterface):
         return None
 
 
-# Ficgs.com
-class InternetGameFicgs(InternetGameInterface):
-    def get_identity(self):
-        return "Ficgs.com", CAT_DL
-
-    def assign_game(self, url):
-        rxp = re.compile(
-            r"^https?:\/\/(\S+\.)?ficgs\.com\/game_(\d+).html", re.IGNORECASE
-        )
-        m = rxp.match(url)
-        if m is not None:
-            gid = str(m.group(2))
-            if gid.isdigit() and gid != "0":
-                self.id = gid
-                return True
-        return False
-
-    def download_game(self):
-        # Check
-        if self.id is None:
-            return None
-
-        # Download
-        return self.download("http://www.ficgs.com/game_%s.pgn" % self.id)
-
-
 # Chessbase
 class InternetGameChessbase(InternetGameInterface):
     def get_identity(self):
@@ -2473,7 +2447,6 @@ chess_providers = [
     InternetGameIdeachess(),
     InternetGameChessdb(),
     InternetGameChesspro(),
-    InternetGameFicgs(),
     InternetGameChessbase(),
     InternetGamePlayok(),
     InternetGamePychess(),

--- a/testing/remotegame.py
+++ b/testing/remotegame.py
@@ -26,7 +26,6 @@ from pychess.Savers.remotegame import (
     InternetGameIdeachess,
     InternetGameChessdb,
     InternetGameChesspro,
-    InternetGameFicgs,
     InternetGameChessbase,
     InternetGamePlayok,
     InternetGamePychess,
@@ -621,18 +620,6 @@ class RemoteGameTestCase(unittest.TestCase):
             ("https://chesspro.ru", False),
         ]  # Not a game (homepage)
         self.executeTest(InternetGameChesspro(), links)
-
-    def testFicgs(self):
-        links = [
-            ("http://FICGS.com/game_95671.html", True),  # Game
-            (
-                "http://www.ficgs.com/game_1234567890.html",
-                False,
-            ),  # Not a game (wrong ID)
-            ("http://www.ficgs.com/view_95671.html", False),  # Not a game (wrong path)
-            ("http://www.ficgs.com", False),
-        ]  # Not a game (homepage)
-        self.executeTest(InternetGameFicgs(), links)
 
     def testChessbase(self):
         links = [

